tidy3d/components/data/sim_data.py

```diff
@@ -36,6 +36,9 @@
 # maps monitor type (string) to the class of the corresponding data
 DATA_TYPE_NAME_MAP = {val.__fields__["monitor"].type_.__name__: val for val in MonitorDataTypes}
 
+# residuals below this are considered good fits for broadband adjoint source creation
+RESIDUAL_CUTOFF_ADJOINT = 0.5
```

Comment:
This seems really large? Or what's the typical scale here?

Reply:
This is a relic of my testing :D but it's not used. I'd say a reasonable number is maybe like 1e-6 but it's not 100% clear.